File: 09 - Py-Js/ques06.py
def indices(mainstr, substr):
    if substr not in mainstr: return -1
    ind = []
    start = 0
    while True:
        index = mainstr.find(substr, start)
        if index == -1: break
        ind.append(index)
        start = index + 1
    return ind
print(indices("abababababababaabab", "abab"))

File: 09 - Py-Js/ques06.js
function indices(mainstr, substr) {
    if (!mainstr.includes(substr)) return -1
    let ind = [];
    let start = 0;
    while (true) {
        let index = mainstr.indexOf(substr, start)
        if (index == -1) break
        ind.push(index)
        start = index + 1
    }
    return ind
}
console.log(indices("ababababab", "ab"))
